testing/citest/spinnaker_testing/spinnaker_test_scenario.py

```diff
@@ -280,25 +282,57 @@ def _initKubeOperationConfigurationParameters(cls, parser, defaults):
         '--kube_credentials',
         dest='spinnaker_kubernetes_account',
         help='DEPRECATED. Replaced by --spinnaker_kubernetes_account')
-    
+
   @classmethod
   def _initAppengineOperationConfigurationParameters(cls, parser, defaults):
     """Initialize arguments for configuring operations for App Engine.
-    
+
+      parser: [argparse.ArgumentParser]
+      defaults: [dict] Default binding value overrides.
+         This is used to initialize the default commandline parameters.
+    """
+    parser.add_argument(
+        '--spinnaker_appengine_account',
+        default=defaults.get('SPINNAKER_APPENGINE_ACCOUNT', None),
+        help='Spinnaker account name to use for test operations against'
+             'App Engine. Only used when managing resources on App Engine.')
+
+  @classmethod
+  def _initOpenStackOperationConfigurationParameters(cls, parser, defaults):
+    """Initialize arguments for configuring operations for OpenStack.
+
       parser: [argparse.ArgumentParser]
       defaults: [dict] Default binding value overrides.
-         This is used to initialize the default commandline parameters.      
+         This is used to initialize the default commandline parameters.
     """
+    # pylint: disable=line-too-long
+    parser.add_argument(
+        '--spinnaker_os_account',
+        default=defaults.get('SPINNAKER_OS_ACCOUNT', None),
+        help='Spinnaker account name to use for test operations against OpenStack.'
+             'Only used when managing resources on OpenStack.')
+    parser.add_argument(
+        '--os_region_name',
+        default=defaults.get('OS_REGION_NAME', None),
+        help='The OpenStack region to test generated instances in (when managing'
+             ' OpenStack). ')
+    parser.add_argument(
+        '--test_os_username',
+        default=defaults.get('TEST_OS_USERNAME', 'my-openstack-account'),
+        help='The OpenStack authentiaction username for test operations against OpenStack. ')
+    parser.add_argument(
+        '--test_os_project_name',
+        default=defaults.get('TEST_OS_PROJECT_NAME', ''),
+        help='OpenStack project to test instances in (when managing OS).')
+
     parser.add_argument(
-      '--spinnaker_appengine_account',
-      default=defaults.get('SPINNAKER_APPENGINE_ACCOUNT', None),
-      help='Spinnaker account name to use for test operations against'
-           'App Engine. Only used when managing resources on App Engine.')
+        '--test_os_auth_url', default=defaults.get('TEST_OS_AUTH_URL', "http://localhost:5000/v2.0"),
```

Comment:
What's the difference between 2 and 3?
In the test server emily set up, we were using 3.